src/proto_bcd/comparator/enum_value_comparator.py

```diff
@@ -51,14 +51,14 @@ def compare(self):
                 context=self.context,
                 change_type=ChangeType.MAJOR,
             )
-        # 3. If both EnumValueDescriptors are existing, check if the name is identical.
-        elif self.enum_value_original.name != self.enum_value_update.name:
+        # 3. If both EnumValueDescriptors are existing, check if the number is identical.
+        elif self.enum_value_original.number != self.enum_value_update.number:
             self.finding_container.add_finding(
-                category=FindingCategory.ENUM_VALUE_NAME_CHANGE,
+                category=FindingCategory.ENUM_VALUE_NUMBER_CHANGE,
                 proto_file_name=self.enum_value_update.proto_file_name,
                 source_code_line=self.enum_value_update.source_code_line,
-                subject=self.enum_value_update.name,
-                oldsubject=self.enum_value_original.name,
+                subject=self.enum_value_update.number,
```

Comment:
I wonder if we should pass `str(self.enum_value_update.number)` just in case. I think it will probably work without `str()` because in `Finding.get_message` [here](https://github.com/googleapis/proto-breaking-change-detector/blob/9674398a9a44cf14e91d5b322db2b3aaaecc288c/src/proto_bcd/findings/finding.py#L67-L69) we just use `format()` and it should accept numbers for `{subject}` and `{oldsubject}`, but I would still convert them to strings just in case.

Reply:
I changed them to include both name and number.